packages/components/src/components/empty-state/empty-state.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Button from '../button';
+import Icon from '../icon';
+import Text from '../text';
+import './empty-state.scss';
+
+type TAction = {
+    label: string;
+    onClick?: VoidFunction;
+    primary?: boolean;
+    tertiary?: boolean;
+    disabled?: boolean;
+};
+
+export type TProps = {
+    icon?: string;
+    action?: TAction;
+} & RequireAtLeastOne<{ title: React.ReactNode; description: React.ReactNode }>;
+
+const EmptyState: React.FC<TProps> = ({ icon, title, description, action }) => (
+    <div className='empty-state'>
+        {icon && <Icon icon={icon} size={128} />}
+        <br />
```

Comment:
Isn't better to use margin instead of `br`? Also, we have `gap` property in the `empty-state` class, isn't that enough for spaces between elements?

Reply:
@hamid-deriv The gap for the icon and button is twice as the rest, So instead of adding more CSS I'm just adding an element to achieve 2 times gap 😁

<img width="731" alt="image" src="https://github.com/binary-com/deriv-app/assets/72082844/ab92e435-8e07-4d77-9cd0-3764cf5c34cf">